src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck is working");
+    }
```

Comment:
```suggestion
public class Truck extends Machine {
    @Override
    public void doWork() {
        System.out.println("Truck is working");
    }
```
![image](https://github.com/mate-academy/jv-oop/assets/20326858/f431de01-90b8-499b-8ff4-62921c761ccd)